package.json

```diff
@@ -3,29 +3,30 @@
   "version": "4.0.4",
   "author": "Johannes Ewald @jhnns",
   "description": "Less loader for webpack. Compiles Less to CSS.",
-  "main": "dist/index.js",
+  "main": "dist/cjs.js",
   "scripts": {
     "create-spec": "babel-node test/helpers/createSpec.js",
     "pretest": "npm run create-spec",
     "test": "jest",
     "posttest": "npm run lint",
-    "travis:test": "yarn run test",
-    "appveyor:test": "npm test",
+    "travis:test": "npm run test -- --runInBand",
+    "appveyor:test": "npm run test",
     "lint": "eslint --cache src test",
     "webpack-defaults": "webpack-defaults",
-    "start": "yarn run build -- -w",
+    "start": "npm run build -- -w",
     "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
     "clean:dist": "del-cli dist",
```

Comment:
Can be removed :), bc of https://github.com/webpack-contrib/less-loader/pull/208/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R30